ceph.spec.in

```diff
@@ -839,7 +839,7 @@ rm -rf %{buildroot}
 %{_sbindir}/ceph-create-keys
 %{_sbindir}/rcceph
 %dir %{_libexecdir}/ceph
-%{_libexecdir}/ceph/ceph_common.sh
+%{_libdir}/ceph/ceph_common.sh
```

Comment:
By the way, I want to change the bug title of http://tracker.ceph.com/issues/18825 to "Generate ceph-osd-prestart.sh paths in upstart/systemd configs at build time", but I failed to see how to do it.